TheMovieDb/Network/Request.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  Request.swift
+//  TheMovieDb
+//
+//  Created by Antonio Hernandez Ambrocio on 01/11/21.
+//
+
+import Foundation
+
+class NetworkManager {
```

Comment:
final, also do we need it as class ? 

https://developer.apple.com/documentation/swift/choosing_between_structures_and_classes